plugins/mcreator-core/datalists/sounds.yaml

```diff
@@ -996,3 +997,188 @@
+- block.dripstone_block.place
+- block.dripstone_block.step
+- block.pointed_dripstone.break
+- block.pointed_dripstone.fall
+- block.pointed_dripstone.hit
+- block.pointed_dripstone.place
+- block.pointed_dripstone.step
+- block.pointed_dripstone.land
+- block.pointed_dripstone.drip_lava
+- block.pointed_dripstone.drip_water
+- block.pointed_dripstone.drip_lava_into_cauldron
+- block.pointed_dripstone.drip_water_into_cauldron
+- block.big_dripleaf.tilt_down
+- block.big_dripleaf.tilt_up
+- block.flowering_azalea.break
+- block.flowering_azalea.fall
+- block.flowering_azalea.hit
+- block.flowering_azalea.place
+- block.flowering_azalea.step
+- block.roots.break
+- block.roots.fall
+- block.roots.hit
+- block.roots.place
+- block.roots.step
+- item.glow_ink_sac.use
+- entity.glow_item_frame.add_item
+- entity.glow_item_frame.break
+- entity.glow_item_frame.place
+- entity.glow_item_frame.remove_item
+- entity.glow_item_frame.place_item
+- entity.glow_squid.ambient
+- entity.glow_squid.death
+- entity.glow_squid.hurt
+- entity.glow_squid.squirt
+- entity.goat.ambient
+- entity.goat.death
+- entity.goat.eat
+- entity.goat.hurt
+- entity.goat.long_jump
+- entity.goat.milk
+- entity.goat.prepare_ram
+- entity.goat.ram_impact
+- entity.screaming.ambient
+- entity.screaming.death
+- entity.screaming.eat
+- entity.screaming.hurt
+- entity.screaming.long_jump
+- entity.screaming.milk
+- entity.screaming.prepare_ram
+- entity.screaming.ram_impact
+- entity.goat.step
+- block.hanging_roots.break
+- block.hanging_roots.fall
+- block.hanging_roots.hit
+- block.hanging_roots.place
+- block.hanging_roots.step
+- block.large_amethyst_bud.break
+- block.large_amethyst_bud.place
+- block.medium_amethyst_bud.break
+- block.medium_amethyst_bud.place
+- block.small_amethyst_bud.break
+- block.small_amethyst_bud.place
+- block.moss_carpet.break
+- block.moss_carpet.fall
+- block.moss_carpet.hit
+- block.moss_carpet.place
+- block.moss_carpet.step
+- block.moss.break
+- block.moss.fall
+- block.moss.hit
+- block.moss.place
+- block.moss.step
+- block.polished_deepslate.break
+- block.polished_deepslate.fall
+- block.polished_deepslate.hit
+- block.polished_deepslate.place
+- block.polished_deepslate.step
+- block.powder_snow.break
+- block.powder_snow.fall
+- block.powder_snow.hit
+- block.powder_snow.place
+- block.powder_snow.step
+- block.rooted_dirt.break
+- block.rooted_dirt.fall
+- block.rooted_dirt.hit
+- block.rooted_dirt.place
+- block.rooted_dirt.step
+- block.sculk_sensor.clicking
+- block.sculk_sensor.clicking_stop
+- block.sculk_sensor.break
+- block.sculk_sensor.fall
+- block.sculk_sensor.hit
+- block.sculk_sensor.place
+- block.sculk_sensor.step
+- block.spore_blossom.break
+- block.spore_blossom.fall
+- block.spore_blossom.hit
+- block.spore_blossom.place
+- block.spore_blossom.step
+- item.spyglass.use
+- item.spyglass.stop_using
+- block.tuff.break
+- block.tuff.fall
+- block.tuff.hit
+- block.tuff.place
+- block.tuff.step
```

Comment:
A few more items are missing from this list:
- item.axe.scrape
- item.bone_meal.use
- block.candle.ambient
- item.dye.use
- item.ink_sac.use
- entity.minecart.inside.underwater
- entity.skeleton.converted_to_stray
- block.small_dripleaf.break
- block.small_dripleaf.fall
- block.small_dripleaf.hit
- block.small_dripleaf.place
- block.small_dripleaf.step
- block.vine.break
- block.vine.fall
- block.vine.hit
- block.vine.place